README.md

```diff
@@ -62,3 +62,27 @@ Once you've turned up the deployment, you can test it by
 
 -   [making a call](http://clearwater.readthedocs.org/en/latest/Making_your_first_call) - make sure you configure your SIP clients with a proxy, as if it were an All-in-One node
 -   [running the live tests](http://clearwater.readthedocs.org/en/latest/Running_the_live_tests) - again, set the PROXY and ELLIS elements, as if it were an All-in-One node.
+
+## Cleaning Up
+
+If you wish to destroy your deployment either to deploy an upgraded version or to free up resources on your docker host, the following may be helpful:
```

Comment:
"to deploy an upgraded version" - is this our recommended way of upgrading?  Maybe better to say "to redeploy with a different configuration or version"?